### ISSUE TITLE
memcached appears to be mandatory, install fails

### ISSUE THREAD

**Issue by fedorov** [girder/large_image#130]
I am trying to install this plugin on CentOS 6. I was able to install `memcached` and `memcached-devel` using standard package manager, but `large_image` setup complained about missing memcached headers. I then removed memcached, and re-run the setup script, and I am still getting those missing header errors. Is it supposed to work without `memcached`? Instructions say it is optional. Any suggestions how to resolve the install problem?

```
Installed /home/af61/girder_env/lib/python2.7/site-packages/large_image-0.2.0-py2.7.egg
Processing dependencies for large-image==0.2.0
Searching for pylibmc>=1.5.1
Reading https://pypi.python.org/simple/pylibmc/
Downloading https://pypi.python.org/packages/23/f4/3904b7171e61a83eafee0ed3b1b8efe4d3c6ddc05f7ebdff1831cf0e15f1/pylibmc-1.5.1.tar.gz#md5=9077704e34afc8b6c7b0b686ae9579de
Best match: pylibmc 1.5.1
Processing pylibmc-1.5.1.tar.gz
Writing /tmp/easy_install-EN12Mo/pylibmc-1.5.1/setup.cfg
Running pylibmc-1.5.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-EN12Mo/pylibmc-1.5.1/egg-dist-tmp-2MIdfj
warning: no files found matching 'runtests.py'
warning: no files found matching '*.py' under directory 'pylibmc'
In file included from src/_pylibmcmodule.c:34:
src/_pylibmcmodule.h:42:36: error: libmemcached/memcached.h: No such file or directory
```

**Comment by manthey** [comment]
This is a failure of pip to install the pylibmc module (because memcached isn't installed).

If you comment out the `pylibmc` line in `requirements.txt`, this should work without memcached.

pylibmc really shouldn't be in requirements.txt, since it isn't a requirement.

**Comment by fedorov** [comment]
Thanks, commenting out `pylibmc` fixed the installation problem. In the retrospect, it is trivial, but as you said, perhaps it should not be there at all.

**Comment by manthey** [comment]
Resolved in PR #213.
